ViewGenerator/build/ViewGenerator.targets

```diff
@@ -107,7 +107,7 @@
   
   <ItemGroup>
     <VG_Inputs Include="**\*.tsx" Exclude="node_modules\**\*.tsx" />
-    <VG_Inputs Include="**\*.ts" Exclude="node_modules\**\*.ts" />
+    <VG_Inputs Include="**\*.ts" Exclude="node_modules\**\*.ts;**\*.d.ts" />
```

Comment:
shouldnt it be node_modules\**\*.d.ts?

Reply:
MSBuild is capturing generated .d.ts on other folders: 
![image](https://user-images.githubusercontent.com/74782571/113897339-ae616480-97c2-11eb-8955-e49c0b5101d6.png)